uarch-bench.sh

```diff
@@ -4,41 +4,15 @@
 
 set -e
 
-# pass cpu # and get the state of the turbo mode, 0 is enabled, 1 is disabled
-function get_core_turbo_intel {
-	echo $(sudo rdmsr -p${core} 0x1a0 -f 38:38)
-}
-
-# get the turbo state of all cores as a string
-function get_turbo_state_string {
-	local turbo_state
-	for core in $cores; do
-    	turbo_state+="$(get_core_turbo_intel $core) "
-	done
-	echo $turbo_state
-}
-
-
-function is_even {
```

Comment:
Oops, thanks for deleting this, this was just spuriously left over from some testing I was doing :).